cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/CertificateRenewalTest.java

```diff
@@ -131,17 +118,17 @@ public static void after() {
                 .endSpec()
                 .build();
 
-        return reconcileCa(context, kafka, () -> new Date());
+        return reconcileCa(vertx, kafka, Date::new);
     }
 
-    private Future<ArgumentCaptor<Secret>> reconcileCa(VertxTestContext context, Kafka kafka, Supplier<Date> dateSupplier) {
+    private Future<ArgumentCaptor<Secret>> reconcileCa(Vertx vertx, Kafka kafka, Supplier<Date> dateSupplier) {
         ResourceOperatorSupplier supplier = ResourceUtils.supplierWithMocks(false);
         SecretOperator secretOps = supplier.secretOperations;
 
         when(secretOps.list(eq(NAMESPACE), any())).thenAnswer(invocation -> {
             Map<String, String> requiredLabels = ((Labels) invocation.getArgument(1)).toMap();
             return secrets.stream().filter(s -> {
-                Map<String, String> labels = new HashMap(s.getMetadata().getLabels());
+                Map<String, String> labels = s.getMetadata().getLabels();
```

Comment:
The original map returned didn't need to be re-ingested into a HashMap for the work on its label keyset to be allowed.